src/index.html

```diff
@@ -21,7 +21,446 @@
     />
   </head>
 
-  <body>
-    <h1>Catalog</h1>
+  <body class="body">
+    <header class="header">
+      <a
+        href="index.html"
+        class="logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="moyo logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__list_item">
+            <a
+              href="#apple"
+              class="nav__link-is-active"
+            >
+              APPLE
+            </a>
+          </li>
+          <li class="nav__list_item">
+            <a
+              href="#samsung"
+              class="nav__link"
+            >
+              SAMSUNG
+            </a>
+          </li>
+          <li class="nav__list_item">
+            <a
+              href="#smartphones"
+              class="nav__link"
+            >
+              SMARTPHONES
+            </a>
+          </li>
+          <li class="nav__list_item">
+            <a
+              href="#laptops&computers"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              LAPTOPS & COMPUTERS
+            </a>
+          </li>
+          <li class="nav__list_item">
+            <a
+              href="#gadgets"
+              class="nav__link"
+            >
+              GADGETS
+            </a>
+          </li>
+          <li class="nav__list_item">
+            <a
+              href="#tablets"
+              class="nav__link"
+            >
+              TABLETS
+            </a>
+          </li>
+          <li class="nav__list_item">
+            <a
+              href="#photo"
+              class="nav__link"
+            >
+              PHOTO
+            </a>
+          </li>
+          <li class="nav__list_item">
+            <a
+              href="#video"
+              class="nav__link"
+            >
+              VIDEO
+            </a>
+          </li>
```

Comment:
The navigation links are repeated in the commented-out section below. Ensure that only one version is used to avoid redundancy.